#! /bin/sh /usr/share/dpatch/dpatch-run
## 0010_maintainer_mode.dpatch by  <fabbe@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Enable maintainer mode to avoid auto* problems.

@DPATCH@
diff -urNad trunk~/configure.in trunk/configure.in
--- trunk~/configure.in	2006-05-29 22:52:46.000000000 +0300
+++ trunk/configure.in	2006-11-01 23:24:55.000000000 +0200
@@ -62,6 +62,8 @@
 AM_INIT_AUTOMAKE(cyrus-sasl, 2.1.22)
 CMU_INIT_AUTOMAKE
 
+AM_MAINTAINER_MODE
+
 # and include our config dir scripts
 ACLOCAL="$ACLOCAL -I \$(top_srcdir)/config"
 
